plugins/vtex/src/plugin.ts

```diff
@@ -37,14 +31,11 @@ registerCommercePlugin(
     const secretKey = settings.get('secretKey');
     const accessKey = settings.get('accessKey');
     const accountName = settings.get('accountName');
-    const isDev = settings.get('environment') === 'development';
     const environment = 'vtexcommercestable';
 
     const baseUrl = (url: string) => {
       const endUrl = `https://${accountName}.${environment}.com.br/${url}`;
-      return `${
-        isDev ? 'http://localhost:5000' : 'https://builder.io'
-      }/api/v1/proxy-api?url=${encodeURIComponent(endUrl)}`;
+      return `https://builder.io/api/v1/proxy-api?url=${encodeURIComponent(endUrl)}`;
```

Comment:
Can we use `apiRoot` so that this works locally as well?

```suggestion
      return `${appState.config.apiRoot()}/api/v1/proxy-api?url=${encodeURIComponent(endUrl)}`;
```